### PR TITLE
Practice One of Cucumber

### PR DESCRIPTION
**Pull request by DavidVallejos**

### CODE REVIEW

File: David/Cucumber/Day02/Class/cucumber/features/google.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#©Copyright 2016 David Vallejos
+#All rights reserved.
+#@David.vallejos@gmail.com
+
+Feature: Test of Google Main Page
+	Create scenaries for test a main page of the google
+Scenario:
```
Comment (karimeSalomon) [inline David/Cucumber/Day02/Class/cucumber/features/google.feature]: Each scenario should have a representative name

File: David/Cucumber/Day02/Class/cucumber/features/google.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	Given i am on the maing page
+	When i clic on the browser 
+		And put url www.google.com 
+	Then i should see the main page of google
+
+Scenario:
+	Given i search a music
```
Comment (karimeSalomon) [inline David/Cucumber/Day02/Class/cucumber/features/google.feature]: is this a precondition?

File: David/Cucumber/Day02/Class/cucumber/features/step_definitions/step_login.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#Se debe ejecutar con los comandos:
+#C:\Users\ArielVallejos\Documents\BDT\Repositorio\AT02\David\Cucumber\Day02\Class\cucumber>cucumber features\login.feature
+
+Given(/^I select ("Sign in"||"Create and account")$/) do
```
Comment (karimeSalomon) [inline David/Cucumber/Day02/Class/cucumber/features/step_definitions/step_login.rb]: you should use only one bar :
 ("Sign in"|"Create and account")

File: David/Cucumber/Day02/Class/cucumber/features/university.feature [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#©Copyright 2016 David Vallejos
+#All rights reserved.
+#@David.vallejos@gmail.com
+
+Feature: Daily University
+	Create two scenaries from my daily university
+
+Scenario: Take a especific class
+Give pay for the class
```
Comment (karimeSalomon) [inline David/Cucumber/Day02/Class/cucumber/features/university.feature]: Typo...Given

File: David/Cucumber/Day02/Class/cucumber/features/university.feature [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+When i go to the class and pay for that
```
Comment (karimeSalomon) [inline David/Cucumber/Day02/Class/cucumber/features/university.feature]: you alrea pay for it in the pre condition, you shouldn't repeat it in your "When" sentence

File: David/Cucumber/Day02/Class/cucumber/features/university.feature [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+Then i can take the class on mi schedule
+
+Scenario: Take a buss to the university
+Given i have to go to the university
+When i search a buss that go to the university
```
Comment (karimeSalomon) [inline David/Cucumber/Day02/Class/cucumber/features/university.feature]: The same of the previous observation

**Comment by DavidVallejos** [comment]
Tanks,

I send my practice and my corrections

________________________________

**Pull request merged**